mithril-common/src/store/adapter/mod.rs

```diff
@@ -5,6 +5,7 @@
 
 mod jsonfile_store_adapter;
 mod memory_adapter;
+mod sqlite_adapter;
```

Comment:
Add a `pub use sqlite_adapter::SQLiteAdapter`, this will also makes some of the clippy warnings disappear :)